### DISCUSSION TITLE
How to open a window inside a defined workspace?

### DISCUSSION THREAD

**Discussion by thepragmaticmero** [YaLTeR/niri#383] [Q&A]
Something like
`niri msg action open-window-on-workspace workspace-1 "flatpak run org.mozilla.firefox"`
Is that a feature that could be easily implemented?, it could improve productivity on lower-end devices and could spawn a complete workspace while booting up.

Is it on your scope for the project (being a minimalist compositor)?

**Comment by YaLTeR** [comment]
Something like this is planned, but not implemented/designed yet.
